src/Block/BlockServiceManager.php

```diff
@@ -190,20 +191,18 @@ public function validate(ErrorElement $errorElement, BlockInterface $block): voi
     }
 
     /**
-     * @throws \RuntimeException
```

Comment:
This shouldn't be removed

Reply:
Yeah my bad, I restored @throws initial tag and add the other exception